assets/src/apps/delivery/layouts/deck/DeckLayoutFooter.tsx

```diff
@@ -328,6 +333,24 @@ const DeckLayoutFooter: React.FC = () => {
         }
       }
     });
+
+    //when lesson 'variables' were getting update via mutate state, we were not sending the updated values to server
```

Comment:
my recollection was that variables were author time settings only that never changed, not something that was persisted even.

Reply:
I confirmed with **@Mia** and she said -> " I think in most cases we didn't change variable values, but this was not always the case, several older/migrated lessons change the value of variables throughout. Lately, we've started using variables as a tool to help personalize and randomize the lessons.
So, I think proceeding with the PR to enable variable persistence would be beneficial. Unless there are any potential drawbacks or unforeseen consequences"